opta/utils.py

```diff
@@ -172,3 +173,21 @@ def check_opta_file_exists(config_path: str) -> Literal[True]:
             )
         )
     return True
+
+
+def compare_datetime(date_time1: datetime, date_time2: datetime) -> int:
+    """
+    Compares date_time2 w.r.t. date_time1.
+    Note: Replacing the TZ Info of date_time1 with TZ Info of date_time2.
+
+    Returns:
+        int:    0:  when date_time1 is equal to date_time2
+                1:  when date_time1 is less than date_time2
+                -1: when date_time1 is greater than date_time2
+    """
+    date_time1.replace(tzinfo=date_time2.tzinfo)
```

Comment:
you can't compare 2 times with different timezones?

Reply:
method removed.